### PR TITLE
customization files needed for CSC reco objects study

### PR DESCRIPTION
**Pull request by nvoytish**
#### PR description:

We've managed to set up 'customize' files for cmsDriver that will implement the tests we'd like to run on CSC local reco:
- loosening roads for association of rechits with segments
- tightening anode time window from 16BX to 8BX when building rechits

### CODE REVIEW

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
@nvoytish  please edit the title of the PR and replace it with something more meaningful, possibly the same title of the backport PR

File: RecoLocalMuon/CSCSegment/python/customSegmentBuilder.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+def widenRoads(process):
+   if hasattr(process,'cscSegments'):
+      for ps in process.cscSegments.algo_psets[4].algo_psets: ps.enlarge = True
```
Comment (slava77) [inline RecoLocalMuon/CSCSegment/python/customSegmentBuilder.py]: ```return process``` is missing, compared to the 11_0_X version

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/28979	3b3bbbb 
- [supposedly] temporary customization methods are added to validate incoming updates for the CSC local reco. Some of the changes were proposed earlier, but later reverted e.g. in #17741 for the hits and/or made only optional in #25684 for the segments. This PR is needed to allow for some centrally produced samples to be set up easily.
- jenkins build tests pass

**Comment by silviodonato** [comment]
+1

**Pull request merged**